src/Apps/Artist/ArtistApp.tsx

```diff
@@ -35,54 +36,56 @@ export class ArtistApp extends React.Component<ArtistAppProps> {
     const { artist, children } = this.props
 
     return (
-      <HorizontalPadding>
-        <Row>
-          <Col>
-            <ArtistHeader artist={artist} />
-          </Col>
-        </Row>
+      <AppContainer>
```

Comment:
Wraps the root component of our apps with a container that sets a max width. This used to be handled in force and was tech-debt that needed to be resolved.